app/api/v2/entities/activity_with_user.rb

```diff
@@ -3,7 +3,7 @@
 module API::V2
   module Entities
     class ActivityWithUser < Grape::Entity
-      format_with(:iso_timestamp) { |d| d.utc.iso8601 }
+      format_with(:iso_timestamp) { |d| d&.utc&.iso8601 }
```

Comment:
mix in a helper
def date_format(ts)
  if ts.nil?
....
  end
end

Reply:
is it required?
It looks very compact now. And safe operator fits well